package.json

```diff
@@ -27,57 +27,55 @@
   },
   "homepage": "https://github.com/Microsoft/powerbi-visuals-enhancedscatter#readme",
   "dependencies": {
-    "powerbi-models": "^1.12.6",
-    "powerbi-visuals-utils-chartutils": "^3.0.1",
-    "powerbi-visuals-utils-colorutils": "^6.0.0",
-    "powerbi-visuals-utils-dataviewutils": "^6.0.0",
-    "powerbi-visuals-utils-formattingutils": "^6.0.0",
-    "powerbi-visuals-utils-interactivityutils": "^6.0.1",
-    "powerbi-visuals-utils-tooltiputils": "^6.0.0"
+    "d3-array": "^3.2.4",
+    "d3-axis": "^3.0.0",
+    "d3-collection": "^1.0.7",
+    "d3-color": "^3.1.0",
+    "d3-scale": "^4.0.2",
+    "d3-selection": "^3.0.0",
+    "d3-transition": "^3.0.1",
+    "lodash.clone": "^4.5.0",
+    "powerbi-visuals-api": "^5.4.0",
+    "powerbi-visuals-utils-chartutils": "^6.0.1",
+    "powerbi-visuals-utils-colorutils": "^6.0.1",
+    "powerbi-visuals-utils-dataviewutils": "^6.0.1",
+    "powerbi-visuals-utils-formattingutils": "^6.0.1",
+    "powerbi-visuals-utils-interactivityutils": "^6.0.2",
+    "powerbi-visuals-utils-tooltiputils": "^6.0.1"
   },
   "devDependencies": {
-    "@types/d3-array": "^3.0.4",
+    "@types/d3-array": "^3.0.5",
     "@types/d3-axis": "^3.0.2",
     "@types/d3-collection": "^1.0.10",
     "@types/d3-color": "^3.1.0",
     "@types/d3-scale": "^4.0.3",
     "@types/d3-selection": "^3.0.5",
     "@types/d3-transition": "^3.0.3",
-    "@types/jasmine": "^4.3.1",
-    "@types/karma": "^6.3.3",
-    "@types/webpack": "^5.28.0",
-    "@typescript-eslint/eslint-plugin": "^5.55.0",
-    "coveralls": "^3.1.1",
-    "css-loader": "^6.7.3",
-    "d3-array": "^3.2.2",
-    "d3-axis": "^3.0.0",
-    "d3-collection": "^1.0.7",
-    "d3-color": "^3.1.0",
-    "d3-scale": "^4.0.2",
-    "d3-selection": "^3.0.0",
-    "d3-transition": "^3.0.1",
+    "@types/jasmine": "^4.3.4",
+    "@types/karma": "^6.3.4",
+    "@types/webpack": "^5.28.1",
+    "@typescript-eslint/eslint-plugin": "^5.60.0",
+    "css-loader": "^6.8.1",
     "eslint-plugin-powerbi-visuals": "^0.8.1",
-    "jasmine": "^4.6.0",
-    "karma": "^6.4.1",
-    "karma-chrome-launcher": "^3.1.1",
-    "karma-coverage": "^2.2.0",
+    "jasmine": "^5.1.0",
+    "karma": "^6.4.2",
+    "karma-chrome-launcher": "^3.2.0",
+    "karma-coverage": "^2.2.1",
     "karma-jasmine": "^5.1.0",
     "karma-junit-reporter": "^2.0.1",
     "karma-sourcemap-loader": "^0.4.0",
-    "karma-typescript": "^5.5.3",
+    "karma-typescript": "^5.5.4",
     "karma-typescript-preprocessor": "^0.4.0",
     "karma-webpack": "^5.0.0",
-    "less": "^4.1.3",
-    "lodash.clone": "^4.5.0",
+    "less": "^4.2.0",
     "lodash.last": "^3.0.0",
     "lodash.range": "^3.2.0",
```

Comment:
move lodash to dependencies

Reply:
lodash.last & lodash.range packages are only imported and used while testing, and aren't imported in main code files